app/views/container_replicator/show.html.haml

```diff
@@ -1,4 +1,4 @@
-- if @display == 'containers'
+- if ['containers', 'container_groups'].include?(@display)
```

Comment:
Using `%w(containers container_groups).include?(@display)` will probably make rubocop happier. Check if it works.